scripts/reorder_json_keys.py

```diff
@@ -4,6 +4,20 @@
 from recipe_scrapers._utils import get_abstract_methods
 
 KEYS = get_abstract_methods()
+NUTRITION_KEYS_ORDER = [
+    "servingSize",
+    "calories",
+    "fatContent",
+    "saturatedFatContent",
+    "unsaturatedFatContent",
+    "transFatContent",
+    "carbohydrateContent",
+    "sugarContent",
+    "proteinContent",
+    "sodiumContent",
+    "fiberContent",
+    "cholesterolContent",
+]
```

Comment:
For consistency with `KEYS` on the previous line, could we rename this to `NUTRITION_KEYS`, and make it available from `recipe_scrapers._utils`?

Reply:
Done! Refactored both the script and test to use the import from utils